Doc/whatsnew/3.11.rst

```diff
@@ -1884,6 +1884,17 @@ New Features
 Porting to Python 3.11
 ----------------------
 
+* Many macros have been converted to functions in the Python C API,
```

Comment:
```suggestion
* Many macros have been converted to static inline functions in the Python C API,
```

Reply:
Incorporated this into my suggestion making the other changes Victor requested :+1: